.circleci/config.yml

```diff
@@ -128,6 +128,10 @@ workflows:
           name: "no-exception-only-javadoc"
           image-name: *custom_img
           command: "./.ci/no-exception-test.sh no-exception-only-javadoc"
+      - validate-with-maven-script:
+          name: "no-exception-samples-ant"
+          image-name: *custom_img
```

Comment:
This image without ant, we can use for this execution, regular circle CI image.

Reply:
Removed.